org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/contentassist/antlr/AntlrProposalConflictHelper.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2009 itemis AG (http://www.itemis.eu) and others.
```

Comment:
i moved this class out of deprecated as it is still in use for new xtext projects

Reply:
:+1: